cmd/server/customer_search.go

```diff
@@ -101,5 +103,9 @@ func buildSearchQuery(params searchParams) (string, []interface{}) {
 		query += " and lower(email) like ?"
 		args = append(args, "%"+strings.ToLower(params.Email))
 	}
+	if params.Status != "" {
+		query += " and status like ?"
+		args = append(args, "%"+strings.ToLower(params.Status))
```

Comment:
No need to double `toLower` since we do it up above when we create the params.

Reply:
Good catch. I removed them for the all of the query params.